Orm/Xtensive.Orm/Orm/Tracing/TraceInfo.cs

```diff
@@ -0,0 +1,36 @@
+namespace Xtensive.Orm.Tracing
+{
+  /// <summary>
+  /// Query tracing information gathered when Trace() method is called on a LINQ query.
+  /// </summary>
+  public class TraceInfo
```

Comment:
maybe introduce class

```
class Caller {
  string MemberName;
  string FilePath;
  int LineNumber;
}
```

just to avoid `Caller` prefix for all the properties?

or maybe the entire `TraceInfo` can be transformed into `CallerInfo` class? Unless we plan to add any other properties to it.